capstone_project/src/main/java/com/google/sps/servlets/SortServlet.java

```diff
@@ -33,6 +35,14 @@
 public class SortServlet extends HttpServlet {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
```

Comment:
I would consolidate this block of code into a private helper method that can be used by all three classes.

I'd imagine it would take in a HttpServletResponse as input and return whether or not it did the redirection.

Reply:
Done!